pkgs/development/libraries/libtasn1/default.nix

```diff
@@ -14,6 +14,9 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ texinfo perl ];
 
   doCheck = true;
+  preCheck = stdenv.lib.optionalString stdenv.isDarwin ''
+    export DYLD_LIBRARY_PATH=`pwd`/lib/.libs
+  '';
```

Comment:
would you mind change it so we can avoid the mass rebuild on NixOS?

    preCheck =
      if stdenv.isDarwin
      then "export DYLD_LIBRARY_PATH=`pwd`/lib/.libs"
      else null;